pkg/manager/internal.go

```diff
@@ -633,6 +636,7 @@ func (cm *controllerManager) startLeaderElection(ctx context.Context) (err error
 			},
 		},
 		ReleaseOnCancel: cm.leaderElectionReleaseOnCancel,
+		Name:            cm.leaderElectionID,
```

Comment:
TILd `Label values may contain any unicode character`, ref https://prometheus.io/docs/concepts/data_model/  so we don't have to sanitize this